src/tests/multihost/alltests/conftest.py

```diff
@@ -111,6 +111,17 @@ def removesssd():
     return _modifysssd
 
 
+@pytest.fixture(scope='function')
```

Comment:
Make the fixture autouse and You do not have to add it to every test.

Reply:
Done